Apps/frontend/src/components/SettingsButton.svelte

```diff
@@ -26,14 +24,19 @@
     class="control-panel--bottom"
     transition:fly={{ y: panelHeight, opacity: 1 }}
     bind:clientHeight={panelHeight}
+    use:clickOutside
+    on:click-outside={() => ($presetPopupOpen = false)}
     data-cy="expanded-preset-popup"
   >
-    <ChannelConfigPopup {waveElement} {indicatorElement} />
+    <SettingsPopup />
   </nav>
 {/if}
 
 <div>
-  <button id="managePresets" on:click={showPopup} data-cy="preset-config-open-popup"
-    >{LABEL_BUTTON_MANAGE_CHANNEL_CONFIG}</button
-  >
+  <button
+    id="settingsButton"
```

Comment:
Don't think we actually need the `id` any more when we adapt the test to use the `data-cy`